code/modules/mob/mob_movement.dm

```diff
@@ -15,21 +15,27 @@
 			return 1
 	return (!mover.density || !density || lying)
 
+//The byond version of these verbs wait for the next tick before acting.
+//	instant verbs however can run mid tick or even during the time between ticks.
+/client/verb/moveup()
+	set name = ".moveup"
+	set instant = 1
+	Move(get_step(mob, NORTH), NORTH)

```

Comment:
Is this in reference to something like the shuttle turning sideways when it docks at the Syndicate base? That's the only time I see any orientation change like that